docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+| `--name`                       | IngressClass controller name (default "pomerium.io/ingress-controller")
+| `--namespaces`                 | namespaces to watch, or none to watch all namespaces
+| `--prefix`                     | Ingress annotation prefix (default "ingress.pomerium.io")
+| `--shared-secret`              | base64-encoded shared secret for communicating with databroker
+| `--update-status-from-service` | update ingress status from given service status (pomerium-proxy)|
+
+The helm chart exposes a subset of these flags for appropriate customization.
+
+## Usage
+
+### Defining Routes
+
+If you've tested Pomerium using the all-in-one service, you're probably familiar with configuring routes in Pomerium's `config.yaml`. In this environment, each route is defined as an Ingress resource in the Kubernetes API.
+The Ingress Controller will monitor Ingress resources in the cluster, creating a Pomerium route definition for each one.  Policy and other configuration options for the route are set by using annotations starting with `ingress.pomerium.io/`.
+
+Example:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    ingress.pomerium.io/policy: '[{"allow":{"and":[{"email":{"is":"user@yourdomain.com"}}]}}]' # This can also be a yaml block quote
+spec:
+  rules:
+  - host: hello.localhost.pomerium.io
+    http:
+      paths:
+      - backend:
+          service:
+            name: nginx-hello
+            port:
+              name: http
+        path: /
+        pathType: Prefix
+```
+
+Becomes:
+
+```yaml
+routes:
+  - from: https://hello.localhost.pomerium.io
+    to: http://nginx-hello.default.svc.cluster.local
+    policy:
+    - allow:
+        and:
+          - email:
+              is: user@yourdomain.com
+```
+
+::: details Write Policies in YAML
+
+You can also define a route's policies using YAML:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: name
+  annotations:
+    ingress.pomerium.io/policy: |
+      - allow:
+          or:
+            - domain:
+                is: pomerium.com
+```
+
+:::
+
+### Supported Annotations
+
+Most configuration keys in non-Kubernetes deployments can be specified as annotation in an Ingress Resource definition. The format is `ingress.pomerium.io/${OPTION_NAME}`. The expandable list below contains the annotations available, which behave as described in our reference documentation (with links to the appropriate reference documentation).
+
+::: details Pomerium-Standard Annotations
+
+- [`ingress.pomerium.io/allow_any_authenticated_user`]
+- [`ingress.pomerium.io/allow_public_unauthenticated_access`]
+- [`ingress.pomerium.io/allow_websockets`]
+- [`ingress.pomerium.io/allowed_domains`]
+- [`ingress.pomerium.io/allowed_groups`]
+- [`ingress.pomerium.io/allowed_idp_claims`]
+- [`ingress.pomerium.io/allowed_users`]
+- [`ingress.pomerium.io/cors_allow_preflight`]
+- [`ingress.pomerium.io/host_path_regex_rewrite_pattern]
```

Comment:
several of these paths are missing a tick.

Reply:
tic added.